### PR TITLE
Provide a way to generate OSGi manifest using template files

### PR DESCRIPTION
**Pull request by denis-anisimov**
Fix for #4232

### CODE REVIEW

File: flow-data/pom.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <packaging>jar</packaging>
 
     <properties>
+        <javax.validation.version>2.0.0.Final</javax.validation.version>
```
Comment (SomeoneToIgnore) [inline flow-data/pom.xml]: Is it ok that you have the same property in the root `pom.xml` but with different version?

> <javax.validation.version>1.0.0.GA</javax.validation.version>

File: flow-push/pom.xml [hunk 2]
```diff
@@ -52,6 +49,19 @@
                     </execution>
                 </executions>
             </plugin>
+             <plugin>
```
[the remark this answers is not in the document]
Comment (SomeoneToIgnore) [reply flow-push/pom.xml]: Well, why not.

[the remark this answers is not in the document]
Comment (denis-anisimov) [reply flow-push/pom.xml]: Done

[the remark this answers is not in the document]
Comment (denis-anisimov) [reply flow-push/pom.xml]: Done.

File: flow-data/pom.xml (hunk 1 above)
Comment (denis-anisimov) [reply flow-data/pom.xml]: Right, thanks. Copy-paste.

**Pull request merged**